sdk/src/node-polyfill.ts

```diff
@@ -0,0 +1,8 @@
+import "./polyfill/crypto";
+import "./polyfill/fetch";
+import "./polyfill/xmlhttprequest";
+import "./polyfill/worker";
+
+if (!globalThis.self) {
+    (globalThis as any).self = globalThis;
+}
```

Comment:
These polyfills are only ever bundled when the node version is built right?

Reply:
Yes, it's only used for Node, they aren't included at all for the browser.